src/debugger/debugger.ts

```diff
@@ -59,19 +59,43 @@ async function getBinaryPath(): Promise<string | undefined> {
 	}
 }
 
-export async function connectToDebugger(functionName: string) {
+export async function connectToDebugger(
+	functionName: string,
+	moduleName: string,
+	fileName: string,
+) {
 	// The binary that is being referred to here, is the binary present in the cargo artifacts.
 	// It looks like this - kani_concrete_playback_check_estimate_size_14615086421508420155
 	const binaryName = await getBinaryPath();
 
+	let functionParameter: string;
+
+	if (fileName === 'main') {
```

Comment:
What are you trying to do here? should you be checking if the function name is main instead?

Reply:
If the file name is main (or for main.rs), we don't need to pass the file name in the function parameter. In fact, passing it seems to break the unit test.

For example, in this example,

main.rs
``` rust
#[cfg(kani)]
mod verification {
    use super::*;

    #[kani::proof]
    pub fn stretched_rectangle_can_hold_original() {
        let original = Rectangle {
            width: kani::any(),
            height: kani::any(),
        };
        let factor = kani::any();
        kani::assume(0 != original.width);  //< explicit requirements
        kani::assume(0 != original.height); //< explicit requirements
        kani::assume(1 < factor); //< explicit requirements
        if let Some(larger) = original.stretch(factor) {
            assert!(larger.can_hold(&original));
        }
    }
    #[test]
    fn kani_concrete_playback_stretched_rectangle_can_hold_original_15788871033967399270() {
        let concrete_vals: Vec<Vec<u8>> = vec![
            // 4611686018427387904ul
            vec![0, 0, 0, 0, 0, 0, 0, 64],
            // 0ul
            vec![0, 0, 0, 0, 0, 0, 0, 0],
            // 2ul
            vec![2, 0, 0, 0, 0, 0, 0, 0],
        ];
        kani::concrete_playback_run(concrete_vals, stretched_rectangle_can_hold_original);
    }
}
```

the unit test name that needs to be passed is -> `verification::kani_concrete_playback_stretched_rectangle_can_hold_original_15788871033967399270`.

I was trying to create the parameter for such cases where the unit tests have a module but belong to main.rs